tools/populate-owners/main.go

```diff
@@ -93,15 +93,17 @@ func orgRepos(dir string) (orgRepos []*orgRepo, err error) {
 	return orgRepos, err
 }
 
-func (orgRepo *orgRepo) getConfig(dir string) (err error) {
-	path := filepath.Join(dir, orgRepo.Organization, orgRepo.Repository)
-	info, err := os.Stat(path)
-	if err != nil {
-		return err
-	}
+func (orgRepo *orgRepo) getDirectories(dirs ...string) (err error) {
```

Comment:
`get` is a weird verb for a method that has a side effect and returns no `Directories`

Reply:
> `get` is a weird verb...

This ended up landing via #1769.  Feel free to push name fixes if you have something else you'd prefer.